.idea/workspace.xml

```diff
@@ -2,21 +2,138 @@
 <project version="4">
```

Comment:
do we also want to keep this file or remove?

Reply:
i suggest we remove every ide related folders from the projects